hazelcast-sql/src/test/java/com/hazelcast/jet/sql/impl/connector/jdbc/InsertJdbcSqlConnectorTest.java

```diff
@@ -166,4 +172,25 @@ public void insertIntoTableReverseColumnOrder() throws Exception {
         );
     }
 
+    @Test
+    public void insertIntoTableNonDefaultSchema() throws SQLException {
+
```

Comment:
nit: I dislike this empty line (there are more cases in whole PR)

Reply:
I like blank first lines in methods that are more than a few lines long, but I don't have a strong opinion on it so I removed the instances of this from the PR :+1: